content/examples/databricks-cluster-remote.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Option 1 - Connecting to Databricks remotely"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+With this configuration, RStudio Server Pro is installed outside of the Spark
+cluster and allows users to connect to Spark remotely using
+[Databricks Connect](https://docs.databricks.com/dev-tools/databricks-connect.html).
+
+<img src="/images/deployment/databricks/rstudio-databricks-remote.png" width='800px' align='center'/>
+
+This is the recommended configuration because it targets separate environments,
+involves simpler configuration, avoids resource contention, and allows RStudio
+Server Pro to connect to Databricks as well as other remote storage and compute
+resources.
+
+## Advantages and limitations
+
+Advantages:
+
+- RStudio Server Pro will remain functional if Databricks clusters are
+  terminated
+- Provides the ability to comminicate with one or more Databricks clusters as a
+  remote compute resource
+- Avoids resource contention between RStudio Server Pro and Databricks
+
+Limitations:
+
+- Cannot easily or efficiently load data from outside of the Spark cluster
```

Comment:
This may need to be re-worded in light of https://github.com/sparklyr/sparklyr/pull/2282.
Still not an efficient way to load data (`copy_to()` definitely isn't efficient) but it is pretty straightforward.

Reply:
Thanks for linking to that PR and work in progress. I've removed the wording in 18a831a for now, and will stick to listing the limitations listed on the DBC documentation page.